packages/common/core-interfaces/src/jsonable.ts

```diff
@@ -78,16 +79,17 @@ export interface Internal_InterfaceOfJsonableTypesWith<T> {
  * ```typescript
  * function foo<T>(value: Jsonable<T>) { ... }
  * ```
- * @alpha
+ * @public
  */
-export type Jsonable<T, TReplaced = never> = /* test for 'any' */ boolean extends (
+export type Jsonable<T = unknown, TReplaced = never> = /* test for 'any' */ boolean extends (
```

Comment:
We should leave this required. No default. There is no intended use for this as a type on its own. It is always meant to be a filter over some type. There are some uses that are explicitly `Jsonable<unknown>` and we'd like them cleaned up.

Reply:
What is the better way to say "I accept anything that can be processed by JSON.stringify"? (with some caveats, like no circular dependencies, etc.)? Is any / unknown the best we got so far?

Reply:
I think we can have a simple explicit type for that use where we need it.
I think I had an explicit `JsonableType` for that in prior PR, but it wasn't loved. We currently have `JsonableTypeWith<never>`. It is remarked in the `JsonableTypeWith` docs:

> Use `JsonableTypeWith<never>` for just JSON serializable types.